elytron/src/main/java/org/wildfly/extension/elytron/ElytronSubsystemParser11_0.java

```diff
@@ -33,11 +33,6 @@ String getNameSpace() {
         return ElytronExtension.NAMESPACE_11_0;
     }
 
-    @Override
-    protected PersistentResourceXMLDescription getMapperParser() {
-        return new MapperParser().getParser();
```

Comment:
Before, you had updated ElytronSubsystemParser10_0.java (i.e., the previous parser), to update the return value to `return new MapperParser(MapperParser.Version.VERSION_6_0).getParser();`. That change should be removed from that file and added here instead (since now ElytronSubsystemParser11_0.java is the previous parser).

Reply:
In that case, as ElytronSubsystemParser10_0 is also supposed to return version 6, I leave the return value as  `new MapperParser(MapperParser.Version.VERSION_6_0).getParser()` there too, right? Previously, that parser used to return the current version.

Reply:
Ah, just realized that the newly introduced `MapperParser.Version.VERSION_6_0` corresponds to `ElytronSubsystemParser10_0` and is also needed for `ElytronSubsystemParser11_0`. This means it is correct to specify this version in `ElytronSubsystemParser10_0` and to remove the `getMapperParser()` method from `ElytronSubsystemParser11_0`.

To help make things clearer though, I think it would be good to rename `MapperParser.CURRENT` so that it references the actual version instead. That way, the next time we add a new version in `MapperParser`, we won't need to worry about updating an older version of the parser from `CURRENT` to the previous version.

Reply:
So just verify I understood correctly, you're suggesting we rename `MapperParser.CURRENT` to say `MapperParser.VERSION_7_0` and then in the ElytronSubsystemParser12_0 class I instantiate the mapper parser with version 7, as opposed to leaving an empty constructor? That way in the future, we won't have to update ElytronSubsystemParser12_0.

Reply:
Yes, exactly.